src/cpp/sensitivity/include/SensitivityWriter.h

```diff
@@ -4,6 +4,7 @@
 
 #include <filesystem>
 
+#include "SensitivityInputReader.h"
```

Comment:
Same as previously, why is the include necessary ?

Reply:
See answer above